SOLUTIONS-InspectingACandidateList.ipynb

```diff
@@ -0,0 +1,606 @@
+   "source": [
+    "import astropy.visualization as vis\n",
+    "\n",
+    "sdss_image_services = vo.regsearch(keywords=['sloan'], servicetype='image')\n",
+    "sdss_image_services.to_table()['ivoid', 'short_name', 'res_title', 'source_value']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sdss_image_service = sdss_image_services.getrecord(\n",
+    "    np.where(np.isin(sdss_image_services['short_name'], b'SDSS SIAP'))[0][0])\n",
+    "sdss_image_service.short_name"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As a workaround to a bug in the SDSS service, pass `format=''` when searching."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sdss_image_table = sdss_image_service.search(pos=pos, size=0.0, format='', intersect='covers')\n",
+    "len(sdss_image_table['Title'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for sdss_rband_record in sdss_image_table:\n",
+    "    if 'Sloan Digital Sky Survey - Filter r' in sdss_rband_record.title:\n",
+    "        break"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sdss_rband_image = fits.open(sdss_rband_record.getdataurl())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = plt.figure()\n",
+    "\n",
+    "ax = fig.add_subplot(1, 1, 1, projection=WCS(sdss_rband_image[0].header))\n",
+    "\n",
+    "interval = vis.PercentileInterval(99.9)\n",
+    "vmin,vmax = interval.get_limits(sdss_rband_image[0].data)\n",
+    "norm = vis.ImageNormalize(vmin=vmin, vmax=vmax, stretch=vis.LogStretch(1000))\n",
+    "ax.imshow(sdss_rband_image[0].data, cmap = 'gray_r', norm = norm, origin = 'lower')          \n",
+    "ax.scatter(ra, dec, transform=ax.get_transform('fk5'), s=500, edgecolor='red', facecolor='none')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cutout = Cutout2D(sdss_rband_image[0].data, pos, size, wcs=WCS(sdss_rband_image[0].header))\n",
+    "fig = plt.figure()\n",
+    "ax = fig.add_subplot(1, 1, 1, projection=cutout.wcs)\n",
+    "vmin,vmax = interval.get_limits(sdss_rband_image[0].data)\n",
+    "norm = vis.ImageNormalize(vmin=vmin, vmax=vmax, stretch=vis.LogStretch(1000))\n",
+    "ax.imshow(cutout.data, cmap = 'gray_r', norm = norm, origin = 'lower')          \n",
+    "ax.scatter(ra, dec, transform=ax.get_transform('fk5'), s=500, edgecolor='red', facecolor='none')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Pick the first 3 galaxies.\n",
```

Comment:
This cell took a really long time to run for me. Maybe put a warning at the top?